blueprints/jsonapi-adapter/files/__root__/__path__/__name__.js

```diff
@@ -1,17 +1,5 @@
 <%= importStatement %>
-import config from '../config/environment';
 
 export default <%= baseClass %>.extend({
   type: '<%= entity %>',
-
-  url: config.APP.API_PATH + '/<%= resource %>',
```

Comment:
I've put the `url` property back but changed it to include `config.APP.API_HOST`, since otherwise we force the user to modify the url through `fetchUrl`, which is not necessary in most cases.

Thoughts?